support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -443,7 +453,7 @@ ControlSystems:
       AverageTimescaleFraction: 0.25
       Average0thDeriv: true
     Controller:
-      UpdateFraction: 0.2
+      UpdateFraction: 0.3 #Changed from 0.03 to 0.3 to increase run speed
```

Comment:
I tried these changes and got incoming char speeds right away. Changing the size UpdateFraction back to 0.03 seemed to fix it.

Reply:
Oh whoops, changing it back to 0.2 seemed to fix it :)

Reply:
Ok, so I think only this last one needs to be 0.2 - I've run 5 sims with the first 4 UpdateFractions at 0.3 and the last one at 0.2 and those have run smoothly.  :)

Reply:
Whatever you find works, please add a short comment so we remember to what extent these options were tested :)